packages/cli/src/commands/versions/bump.ts

```diff
@@ -173,6 +178,33 @@ export default async (opts: OptionValues) => {
     },
   });
 
+  await runParallelWorkers({
+    parallelismFactor: 4,
+    items: unlocked,
+    async worker({ name }) {
+      let info = (await fetchPackageInfo(name)) as YarnInfoInspectData & {
+        backstage?: { moved: string };
```

Comment:
There should be a type defined by ourselves where we can add this field instead. If not we could add that :grin: